src/main/java/transactionexecutor/TransactionExecutorImpl.java

```diff
@@ -0,0 +1,15 @@
+package transactionexecutor;
+
+import db.Storage;
+import fruittransaction.FruitTransaction;
+import java.util.List;
+
+public class TransactionExecutorImpl implements TransactionExecutor {
+    private Storage objectForStorage = new Storage();
+    private final OperationSupplyImpl operationSupply = new OperationSupplyImpl();
+
+    @Override
+    public void execute(List<FruitTransaction> transactions) {
+        objectForStorage.setStorage(operationSupply.startOperation(transactions));
+    }
```

Comment:
```suggestion
    private final OperationStrategy operationStrategy = new OperationStrategy();

    @Override
    public void execute(List<FruitTransaction> transactions) {
       for(FruitTransaction transaction : transactions) {
           val newQuantity = operationStrategy.getHandler(transaction.getOperation()).handle(transaction);
           Storage.setFruitQuantity(transaction.getFruit(), newQuantity);
       }
    }
```

Implement Strategy pattern to get handler for specific operation. Your executor should look like this